pyomo/contrib/mindtpy/tests/test_mindtpy.py

```diff
@@ -33,18 +33,34 @@
                  "Symbolic differentiation is not available")
 class TestMindtPy(unittest.TestCase):
     """Tests for the MindtPy solver plugin."""
-
+    
     def test_OA_8PP(self):
         """Test the outer approximation decomposition algorithm."""
         with SolverFactory('mindtpy') as opt:
-            model = EightProcessFlowsheet(convex=False)
+            model = EightProcessFlowsheet()
```

Comment:
Let's be explicit here and add argument `convex=True` even if it is already on by default.